compiler/infra/Assert.cpp

```diff
@@ -34,9 +35,11 @@
 #include "env/CompilerEnv.hpp"
 #include "il/ResolvedMethodSymbol.hpp"
 #include "infra/Annotations.hpp"
+#include "infra/ILWalk.hpp"
 #include "ras/Debug.hpp"
 #include "stdarg.h"
 
+#define NUM_INSTRUCTIONS_IN_CONTEXT 11
```

Comment:
would it make sense to allow this to be overridden using feGetEnv for those cases where you don't get quite enough context but you don't want to dump the whole thing b/c it is too large?

Reply:
Agreed. Switched to an env var in eb741f025f91e67c92e694cec054c3d4a69087aa.